README.md

```diff
@@ -56,6 +56,26 @@ Most of Sidekiq's overhead is Redis network I/O.
 ActiveJob adds a notable amount of CPU overhead due to argument deserialization and callbacks.
 Concurrency of 30 was determined experimentally to maximize one CPU without saturating it.
 
+Performance Dragonfly
```

Comment:
@mperham I don't want to get intrusive so feel free to suggest changes or move this around. I also did not provide a comparison with Redis which I can happily add here (we will publish some content related to the comparison)

Reply:
I don't think this content is relevant to 99% of Sidekiq users so I'd prefer to move it to a Dragonfly-specific wiki page, which can be linked from the README.

Reply:
I am not familiar on how to edit a `wiki page`. Is this something I can open a PR or?

Reply:
You’d need to provide the content (the Using Redis page is a good reference or starting point). I’ll create one on Monday.